tests/test_exposure.py

```diff
@@ -70,7 +70,7 @@ class ExposureTestCase(lsst.utils.tests.TestCase):
 
     def setUp(self):
         maskedImage = afwImage.MaskedImageF(inFilePathSmall)
-        maskedImageMD = afwImage.readMetadata(inFilePathSmall)
+        maskedImageMD = lsst.afw.fits.readMetadata(inFilePathSmall)
```

Comment:
In other files you've imported readMetadata itself.  I don't have a strong opinion on which one to use, but I think it should be consistent.

Reply:
The only time I do not use "from lsst.afw.fits import readMetadata" is if the file uses several other symbols from the same library (as for any library). In this case there's only one other symbol so I'll import both. But 2 or 3 is my preferred limit.